beacon-chain/rpc/validator/status_test.go

```diff
@@ -68,8 +69,11 @@ func TestValidatorStatus_Deposited(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Could not get validator status %v", err)
 	}
-	if resp.Status != ethpb.ValidatorStatus_DEPOSITED {
-		t.Errorf("Wanted %v, got %v", ethpb.ValidatorStatus_DEPOSITED, resp.Status)
+	if resp.Status != ethpb.ValidatorStatus_UNKNOWN_STATUS {
```

Comment:
A validator should only be in DEPOSITED state after the beacon chain has processed its deposit. I've changed it to match that and instead include the estimated inclusion slot.

Reply:
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/rpc/beacon/validators_stream.go has some logic and maths to work out if the deposit is pending inclusion, and when it will be included.  That said, it could be improved as it assumes 14 seconds per block rather than looking at the real-world block time.